on Derecho
ml load ncarenv/23.06 ncarcompilers/1 gdal/3.7.1 geos/3.9.1 proj/8.2.1 udunits/2.2.28 rstudio conda
conda activate r-4.3


install.packages(c('plotly','magrittr','dplyr','data.table','tidyr','stringr','lubridate','ggplot2','readr','pracma','sqldf','stringi','tidyverse'))

install.packages(c('soilDB', 'apsimx', 'RSQLite', 'sf', 'spData', 'XML', 'berryFunctions', 'datetime', 'FME', 'rnoaa')




proj: proj/8.2.1
rstudio: rstudio/2023.09.0
gdal: gdal/3.6.4, gdal/3.7.1
geos: geos/3.9.1
udunits: udunits/2.2.28
   
conda

ml load ncarenv-basic/23.09 ncarenv/23.09 gdal/3.7.1 geos/3.9.1 proj/8.2.1 udunits/2.2.28 rstudio conda

ml load ncarenv/23.06  gcc/12.2.0 gdal/3.7.1 geos/3.9.1 proj/8.2.1 udunits/2.2.28 rstudio conda

ml load ncarenv-basic/23.09 gcc gdal geos proj udunits conda rstudio
conda activate r-4.3


/glade/u/home/apdixon
/glade/derecho/scratch/apdixon/national_gwp
/glade/work/apdixon/climate
/glade/work/apdixon/Output_nClimGrid
/glade/work/apdixon/Output_climate

Rscript 000_County_Run_parallel.R

If github bad certificate
git config --global http.sslVerify false
https://stackoverflow.com/questions/37444154/git-clone-fails-server-certificate-verification-failed


when transfering via globus
connect to globus personal https://docs.globus.org/globus-connect-personal/install/linux/
ap@ap:~/Downloads$ cd globusconnectpersonal-3.2.3
ap@ap:~/Downloads/globusconnectpersonal-3.2.3$ ./globusconnectpersonal

Then online globus file manager
https://app.globus.org/file-manager?
find UCAR GLADE, then UCAR_Emory to connect to personal 

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

# consider this for doParallel issues
https://cran.r-project.org/web/packages/doFuture/index.html

# Get directories file sizes
du -sh ./*/


